### DISCUSSION TITLE
Problem with "Individual meeting/Part of a group"

### DISCUSSION THREAD

**Discussion by iloveaaservice** [code4recovery/12-step-meeting-list#1044] [General]
My area has multiple instances of meetings with the same name that are in different districts and different regions.  By default, the meeting name is also the group name.

It is important to my area to be able to select by districts or regions but the only way to add district information is to mark each meeting as "Part of a group."  The problem with this is that the last entered/updated "Contact Information" for a group name then applies to all meetings with that group name.

GSO used to avoid this problem by having a unique index on group name and city.  I do not know if that is still the case with the new database system.

Perhaps TSML should also use name/region as a unique group indentifier.

The only work-around I can think of is to add the region to the group name.  Any other suggestions out there?

**Comment by iloveaaservice** [comment]
Just tried my "work-around".  It changes the group name and district for every meeting with that name.

So I guess to use district information in TSML EVERY meeting name must be unique.

Seems to be a BIG BUG!

**Reply by joshreisner** [reply]
every *group* name must be unique, meetings may have the same name without sharing any properties

**Comment by joshreisner** [comment]
hi @iloveaaservice yes, that's it exactly. group identity is established in TSML by the group name, so if you can append something to the group name, such as the region or some areas use the Fellowship Connection group number, eg `One Day at a Time Group #12345` or `One Day at a Time Group (Valley Forge)`, that will make the group unique.

i don't see why you'd consider that to be a bug, TSML has always worked this way. it could be confusing to users to have multiple groups with exactly the same name, but otherwise different characteristics.
